client/payment-details/dispute-details/index.tsx

```diff
@@ -57,10 +58,17 @@ const DisputeDetails: React.FC< DisputeDetailsProps > = ( { dispute } ) => {
 								/>
 							</>
 						) }
-				</CardBody>
-			</Card>
-		</div>
-	);
+					</CardBody>
+				</Card>
+			</div>
+		);
+	}
+
+	if ( [ 'won', 'lost', 'under_review' ].includes( dispute.status ) ) {
+		return <DisputeFooter dispute={ dispute } />;
+	}
```

Comment:
Was a little confused by this component returning/rendering different kinds of content: `<div><Card>` vs `<CardFooter>`.

That makes me think it knows too much about its container. Could the container (`<PaymentChargeDetails>`?) invoke the components it needs? i.e. move the business logic  for dispute details, dispute status & actions etc out to parent, and all these components just render what's needed.

Reply:
Agreed 👍 I've moved the logic for choosing what content and actions to render to `PaymentSummaryDetails`, I think I'm a bit closer to your recommendation.

I've left `DisputeDetails` component untouched for this PR.

Reply:
Update: renamed `DisputeDetails` to `DisputeAwaitingResponseDetails` to clarify it's intended use/content and distinguish it from `DisputeResolutionFooter`.